packages/bvaughn-architecture-demo/components/console/filters/FilterToggles.module.css

```diff
@@ -29,7 +29,9 @@
 
 .Checkbox {
   margin: 0;
-  margin-right: 0.5ch;
+  margin-right: 0.3rem;
```

Comment:
Can we keep our grid in increments of 0.25 (or 0.125 in a pinch)?

Reply:
Yeah, for sure! Updated things to use local styles that use these increments.